app/models/operatingsystems/coreos.rb

```diff
@@ -54,7 +58,7 @@ def use_release_name?
 
   # Helper text shown next to major version (do not use i18n)
   def major_version_help
-    '2512.3'
+    '2512.3 or set to 0 to use current'
```

Comment:
Just wondering: can we support using `current` as the major version and leave minor empty?

Reply:
Major and minor must be numeric, we have a lot of code expecting that.

Reply:
I suspected as much

Reply:
I also considered a blank value but major must be present, only minor can be blank.

Reply:
https://github.com/theforeman/foreman/pull/8777#discussion_r708986859